pkgs/by-name/ge/gel/package.nix

```
@@ -16,19 +16,19 @@
 }:
 rustPlatform.buildRustPackage rec {
   pname = "gel";
-  version = "7.0.3";
+  version = "7.2.0";
 
   src = fetchFromGitHub {
     owner = "geldata";
     repo = "gel-cli";
     tag = "v${version}";
-    hash = "sha256-QP4LtLgF2OWCsPCFzpLR8k/RetfEevSd8Uv/PciHCwk=";
+    hash = "sha256-dqBXtCLES7k8sj7ZJ8igo/RCnZQyL/MuXRtLgTJ/H8E=";
```

I confirmed this src change in locally.
```suggestion
    hash = "sha256-HqMfReKt1Gl2c7ectgcW514ARCgfNi8PaykvKJUZirY=";
```
also see: https://github.com/NixOS/nixpkgs/pull/400314/checks?check_run_id=40849820314